src/main/java/org/isf/generaldata/ConfigurationProperties.java

```diff
@@ -0,0 +1,117 @@
+package org.isf.generaldata;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class ConfigurationProperties {
+	
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+	private Properties prop;
+	
+	protected ConfigurationProperties(String fileProperties, boolean exitOnFail) {
+		this.prop = loadPropertiesFile(fileProperties, logger, exitOnFail);
+	}
+	
+	public static Properties loadPropertiesFile(String fileProperties, Logger logger) {
+		return loadPropertiesFile(fileProperties, logger, false);
+	}
+
+	private static Properties loadPropertiesFile(String fileProperties, Logger logger, boolean exitOnFail) {
+		Properties prop = new Properties();
+		InputStream in = Thread.currentThread().getContextClassLoader().getResourceAsStream(fileProperties);
+		try {
+			prop.load(in);
+			logger.info("File {} loaded.", fileProperties);
+			in.close();
+		} catch (IOException e) {
+			logger.error(">> {} file not found.", fileProperties);
+			if (exitOnFail) System.exit(1);
+		}
+		return prop;
+	}
+	
+	/**
+	 * Method to retrieve a property
+	 * 
+	 * @param property
+	 * @param defaultValue
```

Comment:
This doesn't appear in the method signature

Reply:
Thanks